setup.cfg

```diff
@@ -78,6 +78,7 @@ extras =
 	send2trash
 	# Enables macOS-specific appearance themes.
 	pyobjc; sys_platform == 'darwin'
+	notify-py
```

Comment:
This entry should go before send2trash and can include a "Enables desktop notifications." comment.